vnext/package.json

```diff
@@ -34,7 +34,7 @@
     "abort-controller": "^3.0.0",
     "anser": "^1.4.9",
     "base64-js": "^1.1.2",
-    "deprecated-react-native-prop-types": "^2.3.0",
```

Comment:
@rectified95 edited this to use 2.3.0 before, but integration tooling will overwrite these, to ensure they match with `facebook/react-native`. It looks like upstream is on 2.1.0 instead of 2.3.0, so either of you will need to make the same edit there as well.

Reply:
Will do!